### DISCUSSION TITLE
userCode is not a function

### DISCUSSION THREAD

**Discussion by B0ras** [vercel/community#2653] [Help]
### Summary

I am trying to create an express application using vercel functions. However when I run this function I get error 500 and the logs say `TypeError: userCode is not a function`

it is a very simple express app since I am experimenting 

./api/index.js
`
import express from "express"

const app = express();
const router = express.Router()

router.get('/api', (req, res) => {
    res.json({
        hello: "hello"
    })

});

app.use("/", router)

export { app }
`

### Steps to Reproduce

1. run vercel dev 
2. Make an http GET request on http://localhost:3000/api

**Comment by amyegan** [comment]
Hi @B0ras! I looks like you forgot to include a link to a public repo that we can use to reproduce the error. Please share a minimal reproduction so we can see what you see and explore the error.

There is a guide with more information on [using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel) that might help you as well.
